packages/roosterjs-content-model/lib/editor/coreApi/createContentModel.ts

```diff
@@ -32,8 +32,8 @@ function internalCreateContentModel(
         ...(option || {}),
     };
 
-    if (core.reuseModel) {
-        context.allowCacheElement = true;
+    if (core.reuseModel && !option?.disableCacheElement) {
```

Comment:
We already copied this value in line 32, so here we just need to set disableCacheElement to true if core.reuseModel is false.

```ts
if (!core.reuseModel) {
  context.disableCacheElement = true;
}
```

In general, cache should only be used when both of the following to be true:
1. core.reuseModel
2. !option.disableCacheElement

Sof if any of them is not true, we should disable cache.